### PR TITLE
Style: Doesn't require DIVS

### PR DESCRIPTION
**Pull request by gonsie**

### CODE REVIEW

**Comment by bmcutler** [comment]
Looks good.  

In testing this I noticed that the point "badges" disappear if the assignment is cancelled (nothing "active").
And also the assignment points disappear from the versions pull down menu for a cancelled assignment.

**Comment by bmcutler** [comment]
@ZishanHuang Can you look this over and see if Elsa has appropriately merged your work?  (See if there are any issues beyond the one I noted above).

File: public/view/homework.php [hunk 1]
```diff
@@ -391,56 +411,49 @@ function version_changed(){
...
     echo 'onsubmit="return check_version_change()"';
     echo'>';
     echo '<input type="hidden" name="csrf_token" value="'.$_SESSION['csrf'].'"" />';
-    echo '<input type="submit" class="btn btn-primary" value="Set Version '.$assignment_version.' as ACTIVE Submission Version"></input>';
+    echo '<input type="submit" class="btn btn-primary" value="Make Version '.$assignment_version.' ACTIVE (Grade This Version)"></input>';
     echo '</form>';
   }
 
-  if ($active_version != 0) {
+  if ($active_version != 0 && $assignment_version == $active_version) {
```
Comment (ZishanHuang) [inline public/view/homework.php]: I think the previous "Cancel Submission" button (now the "Do Not Grade This Version(...)") should be present in every version other than version 0, and it should have text "Do Not Grade This Assignment" or something similar.

**Comment by ZishanHuang** [comment]
Aesthetic comment: It kind of looks strange when there is a large black "Select Submission Version" in the second div but not in the first div, and the "Select Assignment" font smaller than "Select Submission Version".

**Comment by ZishanHuang** [comment]
Just found a bug: If clicked on the link to go to the Elsa page (which goes to an error page now) and go back to the page, a div will show up, saying "Invalid Course" or "Invalid Semester".

**Comment by MasterOdin** [comment]
Things I notice that may have been mentioned before:
1. Points per version should always be displayed regardless of which version is selected (or if it's cancelled).
2. I'm not sure I like the styling for the dropdown as there's no great visual indicator that it is one (other than that it's a white square). There should at the very least be a small unobtrusive arrow on the right side.
3. The dropdown for version seems horizontally squashed to me. I'd probably want more padding on it.
4. Need padding on the right details box when viewing a version of a submission (the details being submission date, days late, etc.)
5. I think the header is going to need some additional work as I think we should a) display the submitty logo somewhere in it, b) the name of the course and c) probably stack the name (doing "Hello Firstname (student)" instead of just "student") and breadcrumbs. It could be that the name is just done within the breadcrumbs so that if there is no name it's just "CSCI1100" and if there is a name (say "Introduction to CS"), then it's "CSCI1100: Introduction to CS".
6. Where would each part of the breadcrumb supposed to lead? Clicking CSCI1100 will take you to the overview page of the course and then f16 would take you to a page that would list all course codes for that semester (regardless if you're registered for the course or not).

**Comment by bmcutler** [comment]
1.  Yes, this is a bug.  
5.  Yes, the labels need work. "submitty f16" -> submitty logo, "csci1100" -> "intro to CS" (course title)
6.  Clicking on submitty logo takes you to a page that lists all of the courses you have access to this semester (and some advertising for submitty!).  Clicking on "intro to CS" takes you to the big overview/index/navigation page.

**Pull request merged**